packages/react-server/src/server/server.ts

```diff
@@ -18,10 +17,11 @@ interface Options {
   port?: number;
   assetPrefix?: string;
   proxy?: boolean;
-  assetName?: string | ValueFromContext<string>;
```

Comment:
Strictly speaking the change to assetName and renderError isn't needed as this is functionally identical, but I figure reusing the existing type is nicer than having to repeat the declarations.